app/models/proxy_config.rb

```diff
@@ -104,7 +104,7 @@ def update_version
 
     # Read the value
     version = config.connection.select_value(config.select(:version)).to_i
-    write_attribute_without_type_cast :version, version
+    write_attribute_without_type_cast 'version', version
```

Comment:
This behavior seems to have been changed here: https://github.com/rails/rails/commit/7834363bbfdb68d734143063932ccc5a8814c97f#diff-f029f28568c577e87cb998285cfdc00c05df84e95e9c28eb115c148e528c2b44L46